src/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py

```diff
@@ -1015,84 +1015,70 @@ def write_recording(
     return nwbfile
 
 
-def get_nspikes(units_table: pynwb.misc.Units, unit_id: int):
-    """Return the number of spikes for chosen unit."""
-    ids = np.array(units_table.id[:])
-    indexes = np.where(ids == unit_id)[0]
-    if not len(indexes):
-        raise ValueError(f"{unit_id} is an invalid unit_id. Valid ids: {ids}.")
-    index = indexes[0]
-    if index == 0:
-        return units_table["spike_times_index"].data[index]
-    else:
-        return units_table["spike_times_index"].data[index] - units_table["spike_times_index"].data[index - 1]
-
-
-def add_units(
+def add_units_table(
     sorting: SpikeInterfaceSorting,
     nwbfile: pynwb.NWBFile,
-    property_descriptions: Optional[dict] = None,
-    skip_properties: Optional[List[str]] = None,
-    skip_features: Optional[List[str]] = None,
-    use_times: bool = True,
-    write_as: str = "units",
-    units_name: str = "units",
-    units_description: str = "Autogenerated by nwb_conversion_tools.",
+    property_descriptions: dict = dict(),
+    skip_properties: Optional[List[str]] = list(),
+    units_table_name: str = "units",
+    unit_table_description: str = "Autogenerated by nwb_conversion_tools.",
+    write_units_table_in_processing_module: bool = False,
```

Comment:
Instead of `write_as`, which isn't very specific or explanatory, or `write_units_table_in_processing_module` which is quite a bit long for a keyword argument, what would you think about about `write_as_processing_module: bool = False`?